clappr/clappr-ios
# Handle MediaControl plugins in Loader

# Goal

To load `MediaControl` plugins through the `Loader`.

# How to test

1. Open the example
2. Play the video
3. You should see fullscreen, time indicator and play/pause button plugins
4. Interact with them and make sure that their behavior is ok
